app/controllers/review.controller.js

```diff
@@ -0,0 +1,159 @@
+const db = require("../models");
+const Review = db.review;
+const User = db.user;
+
+// Create and Save a new Review
+
+exports.create = (req, res) => {
+    // Validate request
+    if (!req.body.review) {
+        res.status(400).send({
+            message: "Content can not be empty!"
+        });
+        return;
+    }
+
+    // Create a Review
+    const review = {
+        review: req.body.review,
+        storyId: req.body.storyId,
+        userId: req.body.userId,
+    };
+
+    // Save Review in the database
+    Review.create(review)
+        .then(data => {
+            res.send(data);
+        })
+        .catch(err => {
+            res.status(500).send({
```

Comment:
should `console.log` about the error before sending a response back to the caller